src/tasks/03_removeFalsyValues.js

```diff
@@ -15,3 +15,15 @@
  */
 
 // <-- тут реализация removeFalsyValues
+export function removeFalsyValues(in_arr)
+{
+    let temp_arr = [];
+    for (let i = 0; i < in_arr.length; i++)
+    {
+        if (Boolean(in_arr[i]) === true)
```

Comment:
В операторе if значение в скобках должно быть boolean, ты привел его вызовом Boolean

Далее уже нет смысла выполнять операцию сравнения.

Как бы результат Boolean() это уже true или false, и получается ты дополнительно сравниваешь true === true, false === true

```suggestion
        if (Boolean(in_arr[i]))
```